lib/geoengineer/environment.rb

```diff
@@ -70,6 +71,25 @@ def resource(type, id, &block)
     resource
   end
 
+  def find_template(type)
```

Comment:
These methods are basically the same in Project. I think we should refactor this to the HasResources